app/models/manageiq/providers/hawkular/middleware_manager/event_catcher_mixin.rb

```diff
@@ -0,0 +1,67 @@
+# We use this to provide easy access to both the runner and specs
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
+  # extend ActiveSupport::Concern
+
+  # TODO: perhaps we need to block certain events.  Another way to do this would be to add a tag on events we want to
+  # suck into MiQ, and just fetch with the tag as criteria
+  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze # some Hash of included events
+  EXCLUDE_EVENTS = {}.freeze # TODO: some Hash of exclusions for the included events
+
+  def event_monitor_handle(start_time = nil)
+    require 'middleware/events/middleware_event_monitor'
+    @event_monitor_handle ||= MiddlewareEventMonitor.new(@ems, start_time)
+  end
+
+  def reset_event_monitor_handle
+    @event_monitor_handle = nil
+  end
+
+  def stop_event_monitor
```

Comment:
this is indeed required, but if its a noop just `reset_event_monitor_handle` and you are fine. 
So no need to `rescue` here. 

this is the _Interface_ 
https://github.com/ManageIQ/manageiq/blob/40c1166594e5985638d970710390e34da85a4782/app/models/manageiq/providers/base_manager/event_catcher/runner.rb#L79-L89